multiversx/mx-chain-proxy-go
# handle swagger ui if CLI flag is set

- add `--start-swagger-ui` CLI flag for serving the Swagger UI as well
- if the flag is set, static serve the Swagger UI at root ( `http(s)://proxy-url` )
- add `openapi.json`

future steps:
- use libs to automatically generate the swagger files based on the endpoints' annotations

Testing procedures:
1. start the proxy without the `--start-swagger-ui` flag. it should work just as before
2. start the proxy with the `--start-swagger-ui` flag. 
2.1. check the rest of the endpoints, should work like before
2.2. check the root of the proxy and validate the Swagger documentation as much as possible
